src/components/FighterInput.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+
+class MyFighterInput extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { fighter: '' };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ fighter: event.target.value });
+  }
+
+  render() {
+    return (
+      <div>
+         <label htmlFor="fighter">Fighter :</label>
+        <input
```

Comment:
fix indentation. did you install eslint extension in your vs code?

Reply:
ok correction and commit done